INSTALLING.rst

```diff
@@ -93,10 +93,20 @@ Install prerequisites
     - MPI (tested with OpenMPI, MVAPICH)
     - cereal >= 1.1 (required when ``ENABLE_MPI=on``)
 
-  - For GPU execution (required when ``ENABLE_CUDA=on``):
+  - For GPU execution (required when ``ENABLE_HIP=on``):
 
     - NVIDIA CUDA Toolkit >= 9.0
 
+    **OR**
+
+    - AMD ROCm >= 2.9
+
+      Required components:
+        - rocfft
+        - rocprim
+        - rocthrust
+        - hipCUB
```

Comment:
Is hipCUB really a required component? Don't we provide it as a module?

Should we also document the patches required to the AMD libraries and tools?

Reply:
only req'd externally if building for AMD, which is now documented.

Reply:
We now document that some components of HOOMD-blue do not function properly with current HIP and include a link to the temporary development branch.